pkg/traceql/test_examples.yaml

```diff
@@ -1,258 +1,257 @@
 # valid queries parse successfully and return nil when calling .validate()
 valid:
   # spanset filters
-  - '{ true }'
-  - '{ !true }'
-  - '{ true && false }'
-  - '{ true || false }'
-  - '{ 1 = 2 }'
-  - '{ 1 != 2 }'
-  - '{ 1 > 2 }'
-  - '{ 1 >= 2 }'
-  - '{ 1 < 2 }'
-  - '{ 1 <= 2 }'
-  - '{ -1 = 2 }'
+  - "{ true }"
+  - "{ !true }"
+  - "{ true && false }"
+  - "{ true || false }"
+  - "{ 1 = 2 }"
+  - "{ 1 != 2 }"
+  - "{ 1 > 2 }"
+  - "{ 1 >= 2 }"
+  - "{ 1 < 2 }"
+  - "{ 1 <= 2 }"
+  - "{ -1 = 2 }"
   - '{ "test" =~ "test" }'
+  - '{ "test" !~ "test" }'
```

Comment:
What was the rationale for the quoting change in this file?   Fine with it if there is a reason, but now there is now a mixture of quoting styles.  TraceQL also supports embedded backticks for strings, and this works for me locally and would let us update the remainders:  
```
"{ `test` =~ `test` }"
```

Reply:
sorry for this, it's cause of my editor config for yaml. I will revert this

Reply:
reverted